nextflow_schema.json

```diff
@@ -542,5 +546,47 @@
         {
             "$ref": "#/definitions/generic_options"
         }
-    ]
+    ],
+    "properties": {
```

Comment:
These parameters could be grouped into a group called nf_gpt_parameters or similar. You can do this with the nf-core schema bulild drag and drop interface

Reply:
Thank you, i forgot to update the `nextflow_schema.json` file after my last commit. I now added some groupings to the parameters. 
I added the `gpt_interpretation` parameter a while ago in `screening_parameters`. It controls whether or not the gpt is run in the first place as part of the screening process. I personally think it should stay in "screening_parameters" or do you think i should add it to the new group with all the other nf-gpt related parameters?

Reply:
looks good as it is now, thanks!